data/json/items/comestibles.json

```diff
@@ -4215,6 +4215,26 @@
     "charges": 4,
     "fun": -10
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "coffee_pod",
+    "name": "coffee pods",
+    "name_plural": "coffee pod",
+    "category": "other",
+    "weight": 26,
+    "color": "light_green",
+    "comestible_type": "FOOD",
+    "symbol": "%",
```

Comment:
Consider adding 
```
"looks_like": "hickory_nuts",
```
so that it will use the hickory nuts tile in tilesets that don't have a coffee_pod tile. Or some other item id that you think a coffee pod would look like in most tilesets.